.mocharc.js

```diff
@@ -2,5 +2,5 @@ const path = require('path');
 
 module.exports = {
   spec: path.join(__dirname, 'packages/**/test/**/**/**/*.spec.js'),
-  timeout: 30000
+  timeout: 90000
```

Comment:
So had to bump this because now after adding support for walking all "exports", it literally take this long [to walk lodash](https://github.com/ProjectEvergreen/greenwood/blob/v0.15.3/packages/cli/test/cases/build.default.import-node-modules/build.default.import-node-modules.spec.js#L184) 😳 
```sh
...
ExportNamedDeclaration??????? ./toLength.js
ExportNamedDeclaration??????? ./toNumber.js
ExportNamedDeclaration??????? ./toPlainObject.js
ExportNamedDeclaration??????? ./toSafeInteger.js
ExportNamedDeclaration??????? ./toString.js
ExportNamedDeclaration??????? ./random.js
ExportNamedDeclaration??????? ./wrapperAt.js
ExportNamedDeclaration??????? ./wrapperLodash.js
ExportNamedDeclaration??????? ./plant.js
ExportNamedDeclaration??????? ./wrapperReverse.js
ExportNamedDeclaration??????? ./tap.js
ExportNamedDeclaration??????? ./thru.js
ExportNamedDeclaration??????? ./toIterator.js
ExportNamedDeclaration??????? ./toJSON.js
ExportNamedDeclaration??????? ./wrapperValue.js
ExportNamedDeclaration??????? ./wrapperValue.js
ExportNamedDeclaration??????? ./valueOf.js
ExportNamedDeclaration??????? ./wrapperValue.js
ExportNamedDeclaration??????? ./wrapperChain.js
ExportNamedDeclaration??????? ./seq.default.js
ExportNamedDeclaration??????? ./wrapperValue.js
ExportNamedDeclaration??????? ./wrapperValue.js
ExportNamedDeclaration??????? ./split.js
ExportNamedDeclaration??????? ./startCase.js
ExportNamedDeclaration??????? ./template.js
ExportNamedDeclaration??????? ./templateSettings.js
ExportNamedDeclaration??????? ./toLower.js
ExportNamedDeclaration??????? ./toUpper.js
ExportNamedDeclaration??????? ./trim.js
ExportNamedDeclaration??????? ./trimEnd.js
ExportNamedDeclaration??????? ./trimStart.js
ExportNamedDeclaration??????? ./truncate.js
ExportNamedDeclaration??????? ./unescape.js
ExportNamedDeclaration??????? ./upperCase.js
ExportNamedDeclaration??????? ./upperFirst.js
ExportNamedDeclaration??????? ./words.js
ExportNamedDeclaration??????? ./util.default.js
ExportNamedDeclaration??????? ./connect-mixin.js
ExportNamedDeclaration??????? ./lazy-reducer-enhancer.js
ExportNamedDeclaration??????? ./media-query.js
ExportNamedDeclaration??????? ./metadata.js


  0 passing (30s)
  1 failing

  1) Build Greenwood With:
       Importing packages from node modules
         "before all" hook in "Importing packages from node modules":
     Error: Timeout of 30000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood/packages/cli/test/cases/build.default.import-node-modules/build.default.import-node-modules.spec.js)
      at listOnTimeout (internal/timers.js:554:17)
      at processTimers (internal/timers.js:497:7)
```